com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConverter.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+
+namespace UnityEditor.Rendering.Universal
+{
+    // Might need to change this name before making it public
+    internal abstract class RenderPipelineConverter
+    {
+        /// <summary>
+        /// Name of the converter.
+        /// </summary>
+        public abstract string name { get; }
+
+        /// <summary>
+        /// The information when hovering over the converter.
+        /// </summary>
+        public abstract string info { get; }
+
+        private bool m_IsEnabled = true;
+        /// <summary>
+        /// A check if the converter is enabled or not. Can be used to do a check if prerequisites are met to have it enabled or disabled.
+        /// </summary>
+        public virtual bool IsEnabled { get => m_IsEnabled; set => m_IsEnabled = value; }
```

Comment:
Does this need to have a setter? Feels like that state shouldn't be managed by the converter itself

Reply:
done removed setter.